ppr-ui/src/components/parties/composables/useDebtor.ts

```diff
@@ -115,7 +115,16 @@ export const useDebtor = (props, context) => {
         }
       }
     }
-
+    // format the original address before the compare
+    if (localState.originalDebtor) {
+      localState.originalDebtor.address = formatAddress(localState.originalDebtor.address)
+    }
+    // format the original birthdate the exact same before the compare
+    if (localState.originalDebtor.birthDate && localState.originalDebtor.birthDate.length > 10) {
+      const originalDateOfBirth = new Date(localState.originalDebtor.birthDate)
+      localState.originalDebtor.birthDate = originalDateOfBirth.toLocaleDateString(
+        'en-CA', { timeZone: 'America/Vancouver' }).substring(0, 10) + 'T00:00:00-08:00'
```

Comment:
Oh no, if you do this I think you overwrite the timezone incorrectly still. Won't -08:00 put this in a different timezone when we go back to only 7 hours behind utc? I think you can either convert it to a utc timestamp or get the offset with `getTimezoneOffset`

Reply:
It's the original date of birth, so doesn't actually affect anything... it's just for comparison vs the date of birth in component, just so it doesn't appear that it changed, when it hasn't. Does that make sense?

Reply:
Yeah definitely. So where is the original datetime set then? If it's the api and the api sets it correctly (i.e. -07:00 when the timezone changes) won't this register as a change? Otherwise never mind haha

Reply:
Sorry for dragging this out haha -- I am going to approve and let you make the call since you're more familiar with this piece